libs/blocks/table/table.css

```diff
@@ -264,6 +264,7 @@
 
 .table .section-row .col {
```

Comment:
Is there a reason why this selector `.table .section-row .col` is repeated 2 times in the same file? (L42 and L260)

Reply:
It's a good point, but not something I'd want to switch right now. The file has dozens of CSS Lint issues and would ideally be refactored as a whole, as it had many interventions from different teams. I 100% agree with you, but the problem is not localized to just this declaration unfortunately 😢